test_runner/regress/test_broken_timeline.py

```diff
@@ -29,6 +29,8 @@ def test_local_corruption(neon_env_builder: NeonEnvBuilder):
             ".*failed to load metadata.*",
             ".*load failed.*load local timeline.*",
             ".*: layer load failed, assuming permanent failure:.*",
+            ".*failed to get checkpoint bytes.*",
+            ".*failed get control bytes.*",
```

Comment:
You fixed the typo in this error message in the code, it's now "failed to get control bytes". But that means that this allowed_errors line won't do anything.

Reply:
Looks like it is really not produced now.
Will remove and check.